src/sugar3/graphics/progressicon.py

```diff
@@ -89,11 +116,28 @@ def __draw_cb(self, widget, cr):
     def do_get_preferred_width(self):
         width = self._stroke.get_width()
         return (width, width)
+        '''
+        Example:
+        def do_get_preferred_width(self):
+        print("example")
+        return (100, 100)
+          '''
 
     def do_get_preferred_height(self):
         height = self._stroke.get_height()
         return (height, height)
+        '''
+        Example:
+        def do_get_preferred_height(self):
+        print("example")
+        return (100, 100)
+        '''
 
     def update(self, progress):
         self._progress = progress
         self.queue_draw()
+        '''
+        Call update(progress) with the new progress to update the icon.
+        Updates progressicon with progress's value.
+        Example: update(0.9)
+        '''
```

Comment:
Documentation for an API method must be immediately after the method declaration.  Why is this at the end of the method instead of the start?